client/Index.jsx

```diff
@@ -18,13 +18,13 @@ class Index extends React.Component {
       thingsToDo: [],
       sleepingArrangements: [],
       thingsToKnow: [],
-      listingId: window.location.href.split('/')[3] || 0,
+      listingId: window.location.pathname.slice(0, -1),
     };
   }
 
   componentDidMount() {
     axios
-      .get(`/api/description/${this.state.listingId}`)
+      .get(`/api/description${this.state.listingId}`)
       .then(({ data }) => {
         this.setState({
           descriptions: data.description,
```

Comment:
hey does this still work without the '/'?

Reply:
this.state.listingId is "/10" so it will still work since I took out the forward slash from the hardcoded GET. I couldve edited to only keep the number but im lazy :[

Reply:
Lol after codreviwing your file i saw its just a matter of putting an extra 1 --> ...slice(1, -1).